test/team_collection_test.rb

```diff
@@ -32,16 +32,6 @@ def find(team_id_number)
     @teams.find {|team| team.team_id == team_id_number}
   end
 
-  def test_it_can_find_team_info_using_team_id
-    expected = {
```

Comment:
Good catch, we should look through the tests and see if theres any remaining duplicate tests from merging.